tigerpath/majors_and_certificates/majors/ORF_2018.json

```diff
@@ -4,30 +4,29 @@
+  "explanation": "Operations research and financial engineering may be considered as the modern form of a liberal education: modern because it is based on science, mathematics, computing and technology, and liberal in the sense that it provides for broad intellectual development and can lead to many different types of careers. By choosing judiciously from courses in engineering, science, mathematics, economics, public policy, and liberal arts, each student may design a program adapted to his or her particular interests.\n\nAll students start from a common academic core consisting of statistics, probability and stochastic processes, and optimization. Related courses focus on developing computer skills and exposing students to applications in areas such as finance, operations, transportation, and logistics. Students augment the core program with a coherent sequence of departmental electives. Students may also design specialized programs in areas such as medicine and neuroscience, which must be reviewed and approved by their academic adviser and the departmental representative. Students often draw on courses from economics, computer science, applied mathematics, civil and environmental engineering, mechanical engineering, chemistry, molecular biology, psychology, and the Woodrow Wilson School of Public and International Affairs. Requirements for study in the department follow the general requirements for the School of Engineering and Applied Science and the University.",
```

Comment:
```suggestion
  "explanation": "Operations research and financial engineering may be considered as the modern form of a liberal education: modern because it is based on science, mathematics, computing and technology, and liberal in the sense that it provides for broad intellectual development and can lead to many different types of careers. By choosing judiciously from courses in engineering, science, mathematics, economics, public policy, and liberal arts, each student may design a program adapted to his or her particular interests.\n\nAll students start from a common academic core consisting of statistics, probability and stochastic processes, and optimization. Related courses focus on developing computer skills and exposing students to applications in areas such as finance, operations, transportation, and logistics. Students augment the core program with a coherent sequence of departmental electives. Students may also design specialized programs in areas such as medicine and neuroscience, which must be reviewed and approved by their academic adviser and the departmental representative. Students often draw on courses from economics, computer science, applied mathematics, civil and environmental engineering, mechanical engineering, chemistry, molecular biology, psychology, and public and international affairs. Requirements for study in the department follow the general requirements for the School of Engineering and Applied Science and the University.",
```